A36-xds-for-servers.md

```diff
@@ -183,6 +183,14 @@ HttpConnectionManager support is required. HttpConnectionManager must have valid
 RouteConfiguration (via `route_config` or indirect fields like `rds`) is not
 used nor validated at this time, but may be in the future.
 
+If the `envoy.extensions.filters.http.router.v3.Router` is not present in
```

Comment:
Might be a good idea to explicitly state that this is something that will probably change in the future when we add support for RouteConfigurations on the server side.

Reply:
Good point. I added "implementation isn't using RouteConfiguration" to the list of conditions necessary to ignore the fail-everything filter.